ddl/ddl.go

```diff
@@ -394,9 +395,18 @@ func (d *ddl) close() {
 
 // GetLease implements DDL.GetLease interface.
 func (d *ddl) GetLease() time.Duration {
-	d.m.RLock()
+	// There is a possible dead lock caused by:
```

Comment:
L398-405 can be moved to PR description and be removed from the source code.

Reply:
good